### PR TITLE
OpenShift Pipelines document attributes

### PR DESCRIPTION
**Pull request by nkakkar81**
This PR adds a attribute file to be used in all OpenShift Pipelines assemblies.

### CODE REVIEW

**Comment by nkakkar81** [comment]
@bmcelvee - I have modified the attributes file to be similar to other attribute files I have seen for other buckets. Please review. Thanks!!

**Comment by bmcelvee** [comment]
LGTM

**Comment by bmcelvee** [comment]
@vikram-redhat would you mind reviewing this PR too please? I just want to make sure I'm not missing anything. Thanks!

File: modules/op-document-attributes.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+:op-title: OpenShift Pipelines
+:op-shortname: Pipelines
+:op-ver: 1.0
+//
+// Documentation publishing attributes used in the master-docinfo.xml file
+// Note that the DocInfoProductName generates the URL for the product page.
+// Changing the value changes the generated URL.
+//
+:DocInfoProductName: OpenShift Pipelines
+//
```
Comment (vikram-redhat) [inline modules/op-document-attributes.adoc:19]: Book names are not required. You can remove this section

File: modules/op-document-attributes.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+:op-title: OpenShift Pipelines
+:op-shortname: Pipelines
+:op-ver: 1.0
+//
+// Documentation publishing attributes used in the master-docinfo.xml file
+// Note that the DocInfoProductName generates the URL for the product page.
+// Changing the value changes the generated URL.
+//
+:DocInfoProductName: OpenShift Pipelines
```
Comment (vikram-redhat) [inline modules/op-document-attributes.adoc:18]: This is also not required.

**Comment by nkakkar81** [comment]
@vikram-redhat - All comments are incorporated. Request you to please review. Thanks :)

**Comment by nkakkar81** [comment]
Hi @bmcelvee, I have incorporated the comments from @vikram-redhat, and have received his lgtm on slack. Request you to please merge this PR. Thank you :)

**Comment by bmcelvee** [comment]
No QE review required since this PR is updating the attributes file.

**Pull request merged**